modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/fragment/renderer/ViewRecycleBinJSPSectionFragmentRenderer.java

```diff
@@ -29,12 +37,38 @@ public String getLabelKey() {
 	protected ViewRecycleBinSectionDisplayContext getDisplayContext(
 		HttpServletRequest httpServletRequest) {
 
-		return new ViewRecycleBinSectionDisplayContext(httpServletRequest);
+		return new ViewRecycleBinSectionDisplayContext(
+			_depotEntryLocalService, _groupLocalService, httpServletRequest,
+			language, _objectDefinitionService,
+			_objectDefinitionSettingLocalService,
+			_objectEntryFolderModelResourcePermission, _portal);
 	}
 
 	@Override
 	protected String getJSPPath() {
 		return "/view_recycle_bin.jsp";
 	}
 
+	@Reference
+	private DepotEntryLocalService _depotEntryLocalService;
+
+	@Reference
+	private GroupLocalService _groupLocalService;
```

Comment:
Can you use `groupLocalService` which is already available in `BaseSectionFragmentRenderer`?